conanfile.py

```diff
@@ -10,11 +10,11 @@ class NaiveTsearchConan(ConanFile):
     homepage = "https://github.com/kulp/naive-tsearch"
     url = "https://github.com/kulp/naive-tsearch"
     license = "MIT"
-    exports_sources = "CMakeLists.txt", "README.md", "*.c", "*.h", "*.in", "*.inc", "LICENSE"
+    exports_sources = "CMakeLists.txt", "README.md", "*.c", "*.h", "*.in", "*.inc", "LICENSE", "tests/**"
```

Comment:
In the future, it would be nice to put changes like this into their own commits, but it is not a big deal.

Reply:
Yeah, I know. Sorry about that.
I am using `naive-tsearch` for adding wide character support to ncurses on MSVC.
I was hacking on both projects so didn't apply much attention to patch cleanness..

Reply:
I thought that was the case. If it was a big deal here I could have waited to merge. Thanks for the patch.